core/src/main/java/org/apache/iceberg/rest/requests/UpdateTableRequest.java

```diff
@@ -56,6 +66,10 @@ public List<MetadataUpdate> updates() {
     return updates != null ? updates : ImmutableList.of();
   }
 
+  public TableIdentifier identifier() {
```

Comment:
I think the parser and tests for this class need to be updated for the new field. We should not drop the identifier when it is used in certain cases.